src/lib/plan-frequency/PlanFrequency.spec.ts

```diff
@@ -0,0 +1,78 @@
+import '@testing-library/jest-dom';
+import { fireEvent, render } from '@testing-library/svelte';
+import PlanFrequency from './PlanFrequency.svelte';
+import * as MockDate from 'mockdate';
+
+it('should show a start date input with today as default value', () => {
+	const today = '09/13/2021';
+	MockDate.set(today);
+	const { getByLabelText } = render(PlanFrequency);
+
+	expect(getByLabelText('Start')).toHaveDisplayValue('2021-09-13');
```

Comment:
this test does not really verify that `formatDateForOutput` is executed

Reply:
type="date" of the input element unfortunately always has its value in iso format. as we dont use a component library i will just leave it like this. (they solve it with a type="text" and handle the value logic themselves)